server/graphql/v2/object/Order.js

```diff
@@ -104,6 +104,13 @@ export const Order = new GraphQLObjectType({
           return { value, currency: order.currency };
         },
       },
+      // needed for recurring contributions work, but we should update to encoded id and write v2 payment method object soon
+      legacyPaymentMethodId: {
```

Comment:
This should be replaced by a `PaymentMethod` object with a `legacyId` property.

Reply:
Also need to double check who has rights to see the PaymentMethod attached to an order. Is that a public info?